lib/qb_integration/services/line.rb

```diff
@@ -32,6 +32,34 @@ def build_lines(account = nil)
         lines
       end
 
+      def build_purchase_order_lines(account, purchase_order)
+        @line_items = purchase_order["line_items"]
+        line_items.each do |line_item|
+          line = Quickbooks::Model::PurchaseLineItem.new
+
+          price = line_item["price"]
+          quantity = line_item["quantity"]
+
+          line.amount = (quantity * price)
+          line.description = line_item["name"]
+
+          line.item_based_expense! do |detail|
```

Comment:
This is good. We'll need to add this too: `AccountBasedExpenseLine`

I'm not sure how we know if the line item is an `item` or an `account`. Maybe we can ask someone from Systum or something

Reply:
I think I could add it based on what the line item has since it there's only `AccountBasedExpenseLine` or `ItemBasedExpenseLineDetail`, probably quantity or unit price. I'll look for an account based expense to test with!